### PR TITLE
Added code to convert dollar to coins with the sub-type of dollars.

### PR DESCRIPTION
**Pull request by rahulbhatt1899**
Added code to convert dollar to coins with the sub-type of dollars: penny, nickel, dime, and quarter.

### CODE REVIEW

**Comment by kana800** [comment]
There seems to be an issue with the program.

This is the output
```
Quarters: 0
Dimes: 0
Nickels: 0
Pennies: 0
```
when we insert `0.58`

File: 1-Beginner/dollars_to_cents/js/main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    let s = "Quarters: " + quarter + "\nDimes: " + dime + "\nNickels: " + nickel + "\nPennies: " + penny;
+    alert(s);
+    return s;
+};
+
+
+function foo()
+{
+  let dollarAmt = document.getElementById("dollar").value;
+  document.getElementById("cents").innerHTML = DollarToCents(dollarAmt);
```
Comment (kana800) [inline 1-Beginner/dollars_to_cents/js/main.js:22]: There is no element with an `id` called `cents` present in the `html` file

**Review by kana800 — CHANGES_REQUESTED** [review]
when tested with $`0.58`, the program gave an output
```
Quarters: 0
Dimes: 0
Nickels: 0
Pennies: 0
```
fix this issue

**Comment by rahulbhatt1899** [comment]
Made the changes based on feedback, check it out.

File: 1-Beginner/dollars_to_cents/js/main.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ function DollarToCents(number)
 function foo()
 {
   let dollarAmt = document.getElementById("dollar").value;
-  document.getElementById("cents").innerHTML = DollarToCents(dollarAmt);
+  document.getElementById("centsOutput").innerHTML = DollarToCents(dollarAmt);
```
Comment (kana800) [inline 1-Beginner/dollars_to_cents/js/main.js:22]: should be `coinsOutput`, also I dont see the point of adding this inside `foo()`.  Maybe add this inside `DollarToCents` before `alert(s)`

**Pull request merged**